packages/emails/src/components/LocationInfo.tsx

```diff
@@ -108,14 +108,13 @@ export function LocationInfo(props: { calEvent: CalendarEvent; t: TFunction }) {
       />
     );
   }
-
   return (
     <Info
       label={t("where")}
       withSpacer
       description={providerName || props.calEvent.location}
       extraInfo={
-        providerName === "Zoom" || providerName === "Google" ? (
+        (providerName === "Zoom" || providerName === "Google") && props.calEvent.requiresConfirmation ? (
```

Comment:
Feels good to only have to update this in one place huh? ;)

Reply:
It does! You're a god - amazing work on the email refactor. Working with this change was a pleasure